docs/content/getting-started/cli.mdx

```diff
@@ -215,16 +211,25 @@ $ fga model get --field id --format json
 When using the CLI, the model ID can be specified as a `--model-id` parameter or as part of the configuration.
 
 ## Import tuples
+
 To import tuples, use the`fga tuple write` command. It has the following parameters:
 
-| Parameter                                            | Description | 
-| -----------------------------------------------------|----------------------------------------
-| --file                                               | Specifies the file name  json, yaml and csv files are supported |
-| --max-tuples-per-write (optional, default=1, max=40) | Maximum number of tuples to send in a single write |
-| --max-parallel-requests (optional, default=4)        | Maximum number of requests to send in parallel. Make it larger if you want to import a large number of tuples faster|
+| Parameter                                            | Description                                                                                                          |
+| ---------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------- |
+| --file                                               | Specifies the file name json, yaml and csv files are supported                                                       |
+| --max-tuples-per-write (optional, default=1, max=40) | Maximum number of tuples to send in a single write                                                                   |
+| --max-parallel-requests (optional, default=4)        | Maximum number of requests to send in parallel. Make it larger if you want to import a large number of tuples faster |
+| --hide-imported-tuples (optional, default=false)     | Hide successful imports from output, useful when importing large datasets                                            |
 
-The CLI returns a list of tuples that were successfully written and a list of tuples that were not, with details of the write failure. If you specify `-max-tuples-per-write` greater than one, an error in one of the tuples implies none of the tuples get written.
+The CLI returns a detailed JSON response that includes:
 
+- `successful`: List of successfully written tuples (hidden when using `--hide-imported-tuples`)
+- `failed`: List of tuples that failed to write, including the error reason
+- `total_count`: Total number of tuples processed in this operation
+- `successful_count`: Number of tuples successfully written
+- `failed_count`: Number of tuples that failed to write
+
+When using `--hide-imported-tuples`, the successful tuples are not included in the output, making it more practical when importing large datasets. Failed tuples are always shown to help identify and fix any issues. If you specify `-max-tuples-per-write` greater than one, an error in one of the tuples implies none of the tuples get written.
```

Comment:
I think this was a pre-existing typo, but probably good to fix now if we're touching
```suggestion
When using `--hide-imported-tuples`, the successful tuples are not included in the output, making it more practical when importing large datasets. Failed tuples are always shown to help identify and fix any issues. If you specify `--max-tuples-per-write` greater than one, an error in one of the tuples implies none of the tuples get written.
```